### ISSUE TITLE
Dev Build - 2015-08-21 with IronPython is stable!

### ISSUE THREAD

**Issue by andygraham** [Microsoft/PTVS#799]
Sorry if this is inappropriate here but I can't find any other place to say it.

Good Job :+1: 

For the first time since I started using PTVS with IronPython it now seems stable. All previous versions up to now, including PTVS 2.2, have suffered random freezes and crashes, seemingly associated with Intellisense. Now it seems that I can work without the cloud of the anticipation of an imminent crash hanging over me - I just hope I haven't spoken too soon! :)

Many thanks to all on the PTVS team. I guess that IronPython users are a very small minority of users of PTVS but the ability to integrate C#, in which I have years of investment and experience, and IronPython in a single solution in VS is liberating. Thanks again.

**Comment by denfromufa** [comment]
Great, I should do something similar. I only come to PTVS issues to report bugs and request features :)

I preferred PythonNET over IronPython for integrating with .NET. IronPython cannot load CPython modules using C-API and does not support Python 3. PythonNET follows IronPython as much as possible (import clr; clr.AddReference...), I wonder if it is feasible to port existing IronPython to PythonNET?

Also we used to have discussion forum in codeplex, which is lost now.

**Comment by andygraham** [comment]
Coincidentally I've just been playing with PythonNet 2.1.0.dev.1 and CPython 2.710 to see if my IronPython Windows Forms based GUI debugger would run on it. In fact all the Windows Forms based code, inheriting from Form, events, properties etc. worked fine without modification.

However I hit a major snag in that PythonNet can only convert primitive types to .NET Objects. This means that I couldn't pass the present stack frame from the trace function onto the debugger GUI thread using Control.Invoke and Action delegates formed from Python methods as this needs an Object array to hold the parameters. I gave up then as although I could get hacky with global variables for the cross thread stuff I was also using some Window Forms Control.Tag properties, which are an Object type, to pass Python variable quantities around to keep track of them as the debugger digs deeper into Python sequences and this limitation broke everything. All this code is Python code by the way, no C# at all except I used VS to lay out the Forms and put their classes in a dll to inherit from in the Python code and so avoid a lot of long winded programmatic configuration.
